packages/scripts/src/local/getGasEthHistoricalPrice.js

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env node
+
+/**
+ * @notice This is an example script demonstrating how to get a historical median gas price at a specific timestamp.
+ * It could be used for example to query a GASETH-TWAP-1Mx1M price before committing a vote for a DVM price request.
+ *
+ * @dev Prequisites: before running, you will need to set a GOOGLE_APPLICATIONS_CREDENTIALS environment variable for a service account.
+ * @dev This service account will need GCP admin or BigQuery permissions. This guide provides further instructions: https://cloud.google.com/docs/authentication/getting-started
+ * @dev How to run: ./scripts/local/getGasEthHistoricalPrice.js --time 1612137600 --identifier GASETH-TWAP-1Mx1M
+ */
+const { BigQuery } = require("@google-cloud/bigquery");
```

Comment:
Not positive, but it looks like these dependencies may be broken.

Reply:
Good point! Thanks for pointing this out. Now I remember that I didn't test this script because it requires special GCP permissions to run. I should have mentioned that in the description.

Reply:
Fixed.